### DISCUSSION TITLE
Domotizzare avvolgibili con Shelly

### DISCUSSION THREAD

**Discussion by federoco** [indomus/forum#2309] [Hardware & Firmware]
Ciao a tutti, 
sto leggendo le vostre guide per domotizzare le tapparelle.
Queste montano motori  SOMFY  LT 50 17 giri. 
I motori sono già installati ed ho fatto lasciare i fili sufficientemente lunghi per posizionare facilmente gli switch dentro i cassoni dell'avvolgibile.
Chiederò aiuto per il montaggio all'elettricista perchè la parte squisitamente elettrica non è il mio forte.
Avrei pensato di usare uno switch come questo:

** _LINK RIMOSSO_ **

oppure uno shelly 2.5.

Ecco le mie domande.
Secondo voi possono conviene orientarsi sugli Shelly? Il prezzo è simile e Shelly mi pare più affidabile come marca.

Ovviamente pensavo di fare prima una prova con una sola tapparella.
Se tutto va a buon fine, estenderei l'integrazione a tutte quelle della casa. Purtroppo non ho la possibilità di avere un motore per un test su un banco di prova, quindi avrei due strade: collegare subito lo switch alla tapparella e farlo riconoscere ad HA, sperando che vada tutto bene.
La seconda strada sarebbe di collegarlo ad una lampada su un banco prova, farlo riconoscere ad HA, e vedere se va. Quindi montarlo nell'avvolgibile e configurarlo ad hoc.
Quale è la via preferibile?

Grazie a chi vorrà rispondere.

Fede

**Comment by snakuzzo** [comment]
I LINK AGLI STORE NON SONO AMMESSI !!!
RIMOSSO

**Comment by federoco** [comment]
Salve,
mi scuso. Nella fretta ho copiato/incollato il link e non il nome dell’oggetto a cui si riferiva. 
È possibile essere riammesso in questo ottimo forum?
Grazie

Federico

**Reply by snakuzzo** [reply]
Riammesso?
Guarda che nessuno ti ha escluso

**Comment by federoco** [comment]
Ah grazie. “Rimosso” era il link, pensavo il mio account. 
😅

Federico
